docs/howto/operator/google/cloud/dataprep.rst

```diff
@@ -33,9 +33,15 @@ Prerequisite Tasks
 Get Jobs For Job Group
 ^^^^^^^^^^^^^^^^^^^^^^
 
-To get information about the batch jobs within a Cloud Dataprep job use:
+To get information about jobs within a Cloud Dataprep job use:
 :class:`~airflow.providers.google.cloud.operators.dataprep.DataprepGetJobsForJobGroupOperator`
 
+To get connection Dataprep with Airflow you need Dataprep token.
+https://clouddataprep.com/documentation/api#section/Authentication
+
+It should be added to the Connection in Airflow in JSON format.
+https://airflow.readthedocs.io/en/stable/howto/connection/index.html#editing-a-connection-with-the-ui
+
```

Comment:
Let's add example how it should look like: `{"token": TOKEN}`

Reply:
Done.

Reply:
Can you point where? I must have missed it

Reply:
From 47 to 52 line.